source/assets/stylesheets/_variables.scss

```diff
@@ -11,6 +11,7 @@ $orange: #FDAD2B;
 $yellow: #FED336;
 
 $black: #000000;
+$very-dark-gray: #333333;
```

Comment:
Do we definitely need this colour? Here's `$very-dark-gray` and `$dark-gray` together. 
![screen shot 2015-10-12 at 10 13 14](https://cloud.githubusercontent.com/assets/885223/10424799/2ee3b4c6-70ca-11e5-9cb8-d004ef73f63a.png)

Just trying to prevent variable names like `gray` becoming out of hand and to something like

```
$grey: #595959;
$darkish-grey: #444444;
$dark-grey: #363636;
$darker-grey: #303030;
$darkest-grey: #292929;
$darkerer-grey: #191919;
$darkerest-grey: #111111;
```